Formula/cbc.rb

```diff
@@ -6,6 +6,11 @@ class Cbc < Formula
   # update to EPL-2.0 on next release
   license "EPL-1.0"
 
+  livecheck do
+    url :stable
+    regex(%r{^(?:releases/)?v?(\d+(?:\.\d+)+)$}i)
```

Comment:
```suggestion
    url "https://github.com/coin-or/Cbc/releases/latest"
    regex(%r{href=.*?/tag/(?:releases%2F)?v?(\d+(?:\.\d+)+)["' >]}i)
```

The GitHub repository marks a "latest" release, so we should be checking that instead.

Reply:
Nice call out! :)